sample/src/main/java/com/deploygate/sample/App.java

```diff
@@ -27,12 +27,10 @@ public void onCreate() {
         // Note that you also need to edit your AndroidManifest.xml to activate customized initializer.
         // Refer the comment on stableReal/AndroidManifest.xml included in this sample.
 
-        DeployGate.install(this, new DeployGateCallback() {
-            // Please note that this callback is called iff you have removed the content provider.
-            // For those who wanna use the content provider, SDK provides DeployGate#registerCallback for your use-case.
-
-            @Override
-            public void onInitialized(boolean isServiceAvailable) {
+        DeployGateSdkConfiguration configuration = new DeployGateSdkConfiguration.Builder()
+            // Please note that these callback is called if you have removed the content provider.
```

Comment:
`iff` is not a typo by the way. It's an abbreviation of `if and only if`.

Reply:
Oh.. reverted this change.